packages/apollo-gateway/src/index.ts

```diff
@@ -465,6 +475,34 @@ export class ApolloGateway implements GraphQLService {
   protected async loadServiceDefinitions(
     config: GatewayConfig,
   ): ReturnType<Experimental_UpdateServiceDefinitions> {
+    const getRemoteConfig = (engineConfig: GraphQLServiceEngineConfig) => {
+      return getServiceDefinitionsFromStorage({
+        graphId: engineConfig.graphId,
+        apiKeyHash: engineConfig.apiKeyHash,
+        graphVariant: engineConfig.graphVariant,
+        federationVersion:
+          (config as ManagedGatewayConfig).federationVersion || 1,
+        fetcher: this.fetcher,
+      });
+    };
```

Comment:
You're suggesting just building an object rather than a `getRemoteConfig` function, correct?

e.g.

```typescript
const options = {
  graphId: engineConfig.graphId,
  apiKeyHash: engineConfig.apiKeyHash,
  graphVariant: engineConfig.graphVariant,
  federationVersion:
    (config as ManagedGatewayConfig).federationVersion || 1,
  fetcher: this.fetcher,
};
```

and then...

```typescript
getServiceDefinitionsFromStorage(options);
```

If so, that is absolutely a change I will make.  The reason this ended up like this was something work-in-progress that I think I massaged out in another way (it may have been related to `federationVersion` being not being present on all the types, but I don't recall.  Will re-confirm and adjust pending your confirmation on the above.

Reply:
That's right! Sorry for not being more clear 🙂

Reply:
No worry.  I think you were clear enough.  Resolved with 2eee9012b9dac4e7d5c0c4e84b69071df5552026

Reply:
@trevor-scheer Ah, I realized now: See my comment on the reversion of this 2eee901 in 7466d4f8a58b9ce4a24e8439ee754b933da27b11 in response to the error within [this build](https://circleci.com/gh/apollographql/apollo-server/63901).